jaeger-zipkin/src/main/java/com/uber/jaeger/senders/zipkin/ZipkinSender.java

```diff
@@ -161,4 +161,31 @@ public int close() throws SenderException {
     }
     return n;
   }
+
+  // serviceName/host is needed on annotations so zipkin can query
+  // see https://github.com/uber/jaeger-client-java/pull/75 for more info
+  Span backFillHostOnAnnotations(Span span) {
+    Endpoint host = null;
+    for (BinaryAnnotation binaryAnnotation : span.getBinary_annotations()) {
```

Comment:
core annotations are the more traditional source for Host/Service, they should be tried before the LC. Could also refactor the search into a method that returns as soon as host is found, instead of continuing looping.